src/vm-repair/azext_vm_repair/scripts/win-mount-encrypted-disk.ps1

```diff
@@ -29,7 +32,8 @@ if ($bekFile)
     $bekFilePath = $bekFile.FullName
     Write-Output "Found $bekFilePath"
```

Comment:
Could you explain me this logic bit? What if there is no bek file and there is no bit locker key. Else is removed. 
Also when bitlocker key is set then this should be null $bekPartition.DriveLetter?

Reply:
I think this is largely fixed in latest push. Added elseif statements for when bitlockerkey is provided, and moved the logic to fetch the bek file into an early if block before the unlock-bitlocker cmd.